python/vyos/qos/base.py

```diff
@@ -163,11 +163,11 @@ def _build_base_qdisc(self, config : dict, cls_id : int):
             default_tc += f' red'
 
             qparams = self._calc_random_detect_queue_params(
-                avg_pkt=dict_search('average_packet', config),
-                max_thr=dict_search('maximum_threshold', config),
+                avg_pkt=dict_search('average_packet', config) or 1024,
```

Comment:
Why not make it an XML dict default value? If usingin XML the user will know what the default is and it's not magically 1024 in the code.

Reply:
These parameters are configurable only for `random-detect` policy:

```
vyos@vyos# set qos policy random-detect foo precedence 0 
Possible completions:
   average-packet       Average packet size (bytes) (default: 1024)
   mark-probability     Mark probability for random detection (default: 10)
   maximum-threshold    Maximum threshold for random detection (default: 18)
   minimum-threshold    Minimum threshold for random detection
   queue-limit          Maximum queue size
```

but when we use `red ` qdisc as an internal queue we don't have these params in config tree but still should use default values

```
vyos@vyos# set qos policy priority-queue foo default  queue-type random-detect 
Possible completions:
  <Enter>       Execute the current command
  
vyos@vyos# set qos policy shaper foo default  queue-type random-detect 
Possible completions:
  <Enter>       Execute the current command
 ...
```

should I make these params configurable in this case? Then I can use default values from XML